lib/dynflow/rails/daemon.rb

```diff
@@ -42,23 +64,19 @@ def run_background(command = 'start', options = {})
         STDOUT.puts("Dynflow Executor: #{command} in progress")
 
         options[:executors_count].times do
-          Daemons.run_proc(options[:process_name],
-                           :multiple => true,
-                           :dir => options[:pid_dir],
-                           :log_dir => options[:log_dir],
-                           :dir_mode => :normal,
-                           :monitor => true,
-                           :log_output => true,
-                           :ARGV => [command]) do |*_args|
-                             begin
-                               ::Logging.reopen
-                               run(options[:rails_root])
-                             rescue => e
-                               STDERR.puts e.message
-                               ::Rails.logger.exception('Failed running Dynflow daemon', e)
-                               exit 1
-                             end
-                           end
+          daemons_class.run_proc(
+            options[:process_name],
+            daemons_options(command, options)
+          ) do |*_args|
+            begin
+              ::Logging.reopen
+              run(options[:rails_root], options)
```

Comment:
This should probably be `options[:foreman_root]`

Also when trying to run `bundle exec ruby bin/dynflow-executor run` I'm getting
```
Dynflow Executor: run in progress
Starting Rails environment
Starting dynflow with the following options: {:foreman_root=>"/home/aruzicka/git/foreman", :process_name=>"dynflow_executor", :pid_dir=>"/home/aruzicka/git/foreman/tmp/pids", :log_dir=>"/home/aruzicka/git/foreman/log", :wait_attempts=>300, :wait_sleep=>1, :executors_count=>1, :memory_limit=>0, :memory_init_delay=>7200, :memory_polling_interval=>60}
Exiting
undefined method `[]' for nil:NilClass
/home/aruzicka/git/dynflow/lib/dynflow/rails/daemon.rb:76:in `rescue in block (2 levels) in run_background': undefined method `exception' for nil:NilClass (NoMethodError)
        from /home/aruzicka/git/dynflow/lib/dynflow/rails/daemon.rb:71:in `block (2 levels) in run_background'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons/application.rb:266:in `block in start_proc'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons/application.rb:275:in `start_proc'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons/application.rb:296:in `start'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons/controller.rb:59:in `run'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons.rb:197:in `block in run_proc'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons/cmdline.rb:92:in `catch_exceptions'
        from /home/aruzicka/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/daemons-1.2.4/lib/daemons.rb:196:in `run_proc'
        from /home/aruzicka/git/dynflow/lib/dynflow/rails/daemon.rb:67:in `block in run_background'
        from /home/aruzicka/git/dynflow/lib/dynflow/rails/daemon.rb:66:in `times'
        from /home/aruzicka/git/dynflow/lib/dynflow/rails/daemon.rb:66:in `run_background'
        from bin/dynflow-executor:70:in `<main>'
```